modules/pricer/src/main/java/com/opengamma/strata/pricer/rate/swap/DiscountingSwapLegPricer.java

```diff
@@ -134,28 +133,25 @@ public CurrencyAmount futureValue(SwapLeg leg, RatesProvider provider) {
    * All the payments periods must be of type {@link RatePaymentPeriod}.
    * Each period must have a fixed rate, no FX reset and no compounding.
    * 
-   * @param fixedLeg  the swap fixed leg
+   * @param leg  the swap leg
    * @param provider  the rates provider
    * @return the Present Value of a Basis Point
    */
-  public double pvbp(SwapLeg fixedLeg, RatesProvider provider) {
+  public double pvbp(SwapLeg leg, RatesProvider provider) {
     double pvbpFixedLeg = 0.0;
-    for (PaymentPeriod period : fixedLeg.expand().getPaymentPeriods()) {
+    for (PaymentPeriod period : leg.expand().getPaymentPeriods()) {
       ArgChecker.isTrue(period instanceof RatePaymentPeriod, "PaymentPeriod must be instance of RatePaymentPeriod");
       pvbpFixedLeg += pvbpPayment((RatePaymentPeriod) period, provider);
```

Comment:
Update the variable name, pvbpFixedLeg. Also update the javadoc, especially line 127.